backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -458,6 +458,21 @@ public void deleteOrganization(Organization org) {
     return getOrganizationRoleClaimsForUser(user);
   }
 
+  public boolean isEmailInGroup(final String groupName, final String email) {
+    final GroupList oktaGroupList = _client.listGroups(groupName, null, null);
+
+    final Optional<Group> optGroup =
+        oktaGroupList.stream().filter(g -> groupName.equals(g.getProfile().getName())).findFirst();
+
+    if (optGroup.isEmpty()) {
+      return false;
+    }
+
+    final String lcEmail = email.toLowerCase();
```

Comment:
we should remove the calls to `toLowerCase` now. This was a hotfix I added https://github.com/CDCgov/prime-simplereport/pull/1119 but is not needed if both email strings are coming from Okta.

This issue I was solving is the emails in application.yml were lower case and the emails in Okta were not